### ISSUE TITLE
I still don't get it

### ISSUE THREAD

**Issue by peteravila** [MicrosoftDocs/powerapps-docs#2089]
I come out of this with, at best, a very muddled understanding of the differences and applications of the three different behaviors. The page at least makes a (forgive me: pitiful) attempt at explaining Date Only and Time Zone Independent. No such attempt is made for User Local.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7267ae82-4d15-035f-fe93-cc66469d3b58
* Version Independent ID: 63c7e837-14dd-d199-9dc9-27cd81fd4050
* Content: [Behavior and format of the Date and Time column in Microsoft Dataverse - Power Apps](https://docs.microsoft.com/en-us/powerapps/maker/data-platform/behavior-format-date-time-field)
* Content Source: [powerapps-docs/maker/data-platform/behavior-format-date-time-field.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/live/powerapps-docs/maker/data-platform/behavior-format-date-time-field.md)
* Service: **powerapps**
* GitHub Login: @Mattp123
* Microsoft Alias: **matp**

**Comment by Mattp123** [comment]
@peteravila - can you give at least one example or more detailed information about what you expected to see in the article?

**Comment by peteravila** [comment]
Hi, Matt. Sorry, I have only a vague memory of this, my apologies. If you send me a link to the article, I'll do my best to give you more information.

Peter Avila

**Comment by Mattp123** [comment]
@peteravila - Here you go: https://docs.microsoft.com/en-us/powerapps/maker/data-platform/behavior-format-date-time-field

Thanks

**Comment by peteravila** [comment]
I think my confusion could have been avoided if for each of the settings, there had been a clear explanation of WHAT IT IS, rather than just how it behaves. For example, under User Local: “This stores a UTC date.” Just say it.

Hope this helps.

**Comment by peteravila** [comment]
In general, there is too much “round-about” explanation and the reader is expected to form conclusions that should be simply spelled out. “User Local is a UTC date that is translated to and from the user’s local time.” Period.  Done. Then examples. As it is, I had to read it a dozen times, comparing what is being said in one section to what is being said in another to finally arrive at the explanation of what the different values mean.

It’s almost as if there is more concern with following a formula, or format, for these documents than there is in just explaining in plain, clear language. Different topics will benefit from different approaches to the explanation. The sense I get is that there is a template you guys use somewhere and the bigger concern is following the template. Sometimes it’s best to throw out the template and just explain the darn thing. Clearly. In plain English.

From: Peter Avila
Sent: Thursday, February 11, 2021 1:19 PM
To: 'MicrosoftDocs/powerapps-docs' <reply@reply.github.com>; MicrosoftDocs/powerapps-docs <powerapps-docs@noreply.github.com>
Cc: Mention <mention@noreply.github.com>
Subject: RE: [MicrosoftDocs/powerapps-docs] I still don't get it (#2089)

I think my confusion could have been avoided if for each of the settings, there had been a clear explanation of WHAT IT IS, rather than just how it behaves. For example, under User Local: “This stores a UTC date.” Just say it.

Hope this helps.

**Comment by Mattp123** [comment]
@peteravila - UTC would be a storage implementation detail, which isn't really relevant for app makers as the Date and Date and Time columns display calendar and or time input UI, and isn't within the scope for a topic about the format (options available in Power Apps) and behavior (zoned or not). 

Closing issue.
